scholarly_content_detail/settings/prod.py

```diff
@@ -0,0 +1,13 @@
+"""
```

Comment:
not production-ready yet of course, but will get there separately (this is generated code)